tests/e2e/common.go

```diff
@@ -313,7 +359,7 @@ func (td *OsmTestData) InstallOSM(instOpts InstallOSMOpts) error {
 		return errors.Wrap(err, "failed to run osm install")
 	}
 
-	return nil
+	return td.WaitForPodsRunningReady(td.osmNamespace, 90*time.Second, 1)
```

Comment:
We should keep the expected number of pods in a variable that gets updated to reflect the options since installing vault adds a pod, cert-manager adds 3, etc.

Reply:
I did not delete the cert-manager wait. Test speciic ones can coexist, while this just ensures at least 1 was seen (which is sort of the default scenario)

Reply:
Would keeping track here eliminate the need to have the test-specific ones (at least for the control plane namespace)? Looks like tests should continue to work the same way as before, so that's what's really important. Changing this could be a future PR if we want that.

Reply:
ok fair enough

Reply:
I'll open an issue to see if we want this addressed differently post-patch (have Install get the waits on variable or else).